lua/entities/gmod_tardis_debug_pointer/shared.lua

```diff
@@ -0,0 +1,36 @@
+-- TARDIS debug pointer
+-- Creators: Brundoob, Parar020100 and RyanM2711
+
+ENT.Type = "anim"
+ENT.Base = "base_gmodentity"
+ 
+ENT.PrintName		= "TARDIS Debug Pointer"
+ENT.Author			= "Brundoob"
+ENT.Purpose			= "Print locations in the TARDIS"
+ENT.Category		= "Doctor Who - TARDIS"
+ENT.Spawnable 		= false
+ENT.AdminOnly		= false
+
+concommand.Add("tardis2_debug_pointer", function(ply,cmd,args)

```

Comment:
We should have some safety around this to ensure people cannot spawn loads of these, maybe checking ply:IsAdmin()?